nbclassic/notebookapp.py

```diff
@@ -107,7 +107,7 @@ class NotebookApp(
     NotebookAppTraits,
 ):
 
-    name = 'nbclassic'
+    name = 'notebook'
```

Comment:
Not sure we want to rename the extension to `notebook`. We may rename `notebookapp`. to `nbclassicapp`

Reply:
If we rename the extension to `notebook`, we will not be able to run simultaneously notebook7 and nbclassic, which is the goal.

Reply:
The extension has been named 'notebook' for a long time. 0.4 [renamed it](https://github.com/jupyter/nbclassic/pull/96/files#diff-baf53b9a1d8f038c7de824e4928d10356271b26bacf19ffccba98454e685438eL109-R110). The rename is only partial, which is why it's unclear to me whether this was intentional, or caught in a find/replace.

Reply:
With the goal to run both notebook7 and nbclassic at the same time, this change was intentional

Reply:
Changing the main app's name to `nbclassic` caused #136.